### ISSUE TITLE
Documentation with html error code 404

### ISSUE THREAD

**Issue by samuelbernardolip** [indigo-dc/jenkins-pipeline-library#123]
The url present in readme.md pointing to JePL documentation is giving 404 html code.
The review was at the date of this issue and related to release/2.1.0.

**Comment by samuelbernardolip** [comment]
For branch release/2.1.0 the readme.md have wrong reference to documentation:
```
<a href="https://indigo-dc.github.io/jenkins-pipeline-library/stable_2.1.0/" rel="nofollow">documentation</a>
```
The correct url is https://indigo-dc.github.io/jenkins-pipeline-library/release/2.1.0/.

**Comment by orviz** [comment]
Fixed directly in `release/2.1.0` branch
